packages/react-pose/src/components/PoseElement/index.tsx

```diff
@@ -273,18 +277,29 @@ class PoseElement extends React.PureComponent<PoseElementInternalProps> {
   }
 
   setPose(pose: CurrentPose) {
-    const { onPoseComplete } = this.props;
     const poseList: string[] = Array.isArray(pose) ? pose : [pose];
 
     Promise.all(poseList.map(key => key && this.poser.set(key))).then(
-      () => onPoseComplete && onPoseComplete(pose)
+      () => this.callCompleteCallbacks(pose)
```

Comment:
I think this might get called at wrong times and for elements which don't use certain poses, u can have a parent which transitions from closed to opened with a child that has no config for `opened` pose and thus I believe the child's onPoseComplete shouldnt get called